Documentation/project-docs/uwp-compat.md

```diff
@@ -0,0 +1,489 @@
+[System.Net.Http] System.Net.Http.HttpClientHandler
+```
+
+# System.Linq
+## Members
+
+```
+M:System.Linq.Expressions.Expression.GetDelegateType(System.Type[])
+M:System.Linq.Expressions.Expression.Lambda(System.Linq.Expressions.Expression,System.String,System.Boolean,System.Collections.Generic.IEnumerable{System.Linq.Expressions.ParameterExpression})
+[System.Linq.Expressions] System.Linq.Expressions.Compiler.DelegateHelpers::MakeNewCustomDelegate(Type[])
+```
+
+# System.IO
+Note: This data was collected from comments on UWP specific disabled tests
+- Accessing drive format is not permitted inside an AppContainer
+- GetDiskFreeSpaceEx blocked in AppContainer
+
+## Namespaces
+
+```
+System.IO.FileSystemAclExtensions
+```
+
+## Members
+
+```
+M:System.IO.Ports.SerialPort.GetPortNames
+[System.IO.Compression] System.IO.Compression.ZLibException::.ctor(SerializationInfo, StreamingContext)
+[System.IO.FileSystem.DriveInfo] System.IO.DriveInfo::.ctor(SerializationInfo, StreamingContext)
+[System.Collections] System.Collections.Generic.LinkedList`1.Enumerator::.ctor(SerializationInfo, StreamingContext)
+```
+
+# Crypto
+## Members
+
+```
+[System.Security.Cryptography.Cng] System.Security.Cryptography.ECDsaCng::SpecialNistAlgorithmToCurveName(string)
+S.R.Extensions
+[System.Runtime.Extensions] System.Collections.Hashtable.SyncHashtable::.ctor(SerializationInfo, StreamingContext)
+```
+
+# SqlClient
+Note: This assembly is marked with UWPCompatible=false
+
+## Members
+```
+[System.Data.SqlClient] System.Data.LocalDBAPI::LoadProcAddress()
+[System.Data.SqlClient] System.Data.SqlClient.SNI.LocalDB::GetLocalDBConnectionString(string)
+```
+
+# DirectoryServices
+Note: This assembly is marked with UWPCompatible=false
+
+Note: This data was collected from comments on UWP specific disabled tests
+- Getting information about domain is denied inside AppContainer
+
+# System.Security.Cryptography.OpenSsl
+Note: This assembly is marked with UWPCompatible=false
+
+# System.Reflection.Context
+Note: This assembly is marked with UWPCompatible=false
+
+# StackFrame
+Note: This data was collected from comments on UWP specific disabled tests
+- [uapaot] StackFrame is not supported
+
+# MemoryMappedFiles
+Note: This data was collected from comments on UWP specific disabled tests
+- Windows API returns IO exception error code when viewAccess is ReadWriteExecute
+
+# System.Threading
+Note: This data was collected from comments on UWP specific disabled tests
+- [uapaot] ThreadPool.SetMinThreads is not supported
+
+# System.Runtime*
+Note: This data was collected from comments on UWP specific disabled tests
+- [uapaot] Assembly.Load(byte[])")] not supported
+- [uap, not uapaot] When getting resources from PRI file the casing doesn't matter, if it is there it will always find and return the resource
+- [uapaot] Non-zero lower-bounded arrays not supported 
+- [uapaot] Exception.TargetSite always returns null
+
+# System.Private.CoreLib
+
+```
```

Comment:
Can you help digest/summarize this list? Eg., for all the serialization ones, you can say that all serialization constructors and ISerializable implementations will throw PNSE for types that aren't marked [Serializable] -- those that are are all listed here: https://docs.microsoft.com/en-us/dotnet/standard/serialization/binary-serialization#binary-serialization-in-net-core

That would make it easier to diagest/rationalize than an API list.

Such comments can be gathered from the OneNote in many cases...